tests/functional_tests/t0_main/mp/06-5-share-child-non-service.yea

```diff
@@ -1,8 +1,11 @@
 id: 0.mp.06-5-share-child-non-service
 plugin:
   - wandb
+env:
+  - WANDB_DISABLE_SERVICE: "true"
```

Comment:
can you add a test for telemetry,,, ie, no '23' in features?

Reply:
ah good idea!